robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf.xacro

```diff
@@ -6,11 +6,12 @@
   <xacro:include filename="$(find robotino_description)/urdf/base.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/hokuyo.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/kinect.urdf.xacro" />
+  <xacro:include filename="$(find robotino_description)/urdf/base_shell.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/pan_tilt_axis.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/arm.urdf.xacro" />
   <xacro:include filename="$(find softhand_description)/model/softhand_withCableCanal.urdf.xacro" />
 
-  <xacro:base name="base" has_8dof="false"/>
+  <xacro:base name="base" has_8dof="true"/>
```

Comment:
an URDF model with the base joints is only needed to generate the moveit configuration https://github.com/squirrel-project/squirrel_robotino/blob/indigo_dev/robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino_base_plan.urdf.xacro , if you load an URDF with 8 dof as robot_description, the urdf model will limit the movements of the base, the problem that reported Senka.